cmd/generate/config/rules/notion.go

```diff
@@ -0,0 +1,38 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func Notion() *config.Rule {
+    // Define the identifiers that match the Keywords
+    identifiers := []string{"ntn", "ntn_", "notion"}
```

Comment:
"ntn" or "notion" aren't necessary because the token has a known prefix:
```suggestion
    // Define the identifiers that match the Keywords
    identifiers := []string{"ntn_"}
```